apmpackage/apm/changelog.yml

```diff
@@ -1,6 +1,8 @@
-# newer versions go on top
-#
-# change type can be one of: enhancement, bugfix, breaking-change
+- version: 8.4.2
```

Comment:
Maybe this PR is too late, since L6 points to `8.4.2` too?

https://github.com/elastic/apm-server/pull/9007 is the one that actually bumped the version. 

@amannocci , those files were not updated for this PR, is that expected?

Reply:
Yes, we add the placeholder entry when we bump the version. Then during the dev cycle the placeholder entry is replaced when something actually changes.